docs/reference/api/pages.md

```diff
@@ -0,0 +1,511 @@
+
+Individual page objects will include `_children` and `_ancestor` arrays, as well as the `rank` and `level` properties to indicate position in the page tree.
+
+```json
+{
+  "results": [
+    {
+      "_id": "ckhdscx5900054z9k88uqs16w",
+      "orphan": false,
+      "visibility": "public",
+      "type": "@apostrophecms/home-page",
+      "title": "Home Pager",
+      "slug": "/",
+      "rank": 0,
+      "level": 0,
+      "path": "ckhdscx5900054z9k88uqs16w",
+      // ...
+      "_url": "http://localhost:3000/",
+      "_ancestors": [],
+      "_children": [
+        "ckhfen8ls0005mq9k8p9izkjt",
+        "ckhurshqd00088v9k6pfnqpjz",
+        "ckhdscx6000084z9k56lybljw"
+      ]
+      },
+    {
+      "_id": "ckhfen8ls0005mq9k8p9izkjt",
+      "visibility": "public",
+      "type": "article-page",
+      "title": "Blog",
+      "slug": "/blog",
+      "rank": 0,
+      "path": "ckhdscx5900054z9k88uqs16w/ckhfen8ls0005mq9k8p9izkjt",
+      "level": 1,
+      // ...
+      "_url": "http://localhost:3000/blog",
+      "_children": []
+    },
+    {
+      "_id": "ckhurshqd00088v9k6pfnqpjz",
+      "visibility": "public",
+      "type": "default-page",
+      "title": "About us",
+      "slug": "/about-us",
+      "rank": 1,
+      "path": "ckhdscx5900054z9k88uqs16w/ckhurshqd00088v9k6pfnqpjz",
+      "level": 1,
+      // ...
+      "_url": "http://localhost:3000/about-us",
+      "_children": [
+        "ckiz0pmsg006gdr9ker7e99yt",
+        "ckiz0pyym0070dr9kv8hzm7y0"
+      ]
+    },
+    {
+      "_id": "ckiz0pmsg006gdr9ker7e99yt",
+      "visibility": "public",
+      "type": "default-page",
+      "title": "Our history",
+      "slug": "/about-us/our-history",
+      "rank": 0,
+      "path": "ckhdscx5900054z9k88uqs16w/ckhurshqd00088v9k6pfnqpjz/ckiz0pmsg006gdr9ker7e99yt",
+      "level": 2,
+      // ...
+      "_url": "http://localhost:3000/about-us/our-history",
+      "_children": []
+    },
+    {
+      "_id": "ckiz0pyym0070dr9kv8hzm7y0",
+      "visibility": "public",
+      "type": "default-page",
+      "title": "Leadership and staff",
+      "slug": "/about-us/leadership-and-staff",
+      "rank": 1,
+      "path": "ckhdscx5900054z9k88uqs16w/ckhurshqd00088v9k6pfnqpjz/ckiz0pyym0070dr9kv8hzm7y0",
+      "level": 2,
+      // ...
+      "_url": "http://localhost:3000/about-us/leadership-and-staff",
+      "_children": []
+    },
+    {
+      "_id": "ckhdscx6000084z9k56lybljw",
+      "visibility": "public",
+      "type": "@apostrophecms/trash",
+      "title": "Trash",
+      "slug": "/trash",
+      // ...
+      "rank": 3,
+      "path": "ckhdscx5900054z9k88uqs16w/ckhdscx6000084z9k56lybljw",
+      "level": 1,
+      "_children": []
+    }
+  ]
+}
+```
+
+## `GET: /api/v1/@apostrophecms/page/:id`
+
+### Request example
+
+```javascript
+// Request inside an async function.
+const document = await fetch('http://example.net/api/v1/@apostrophecms/page/ckitdo5oq004pu69kr6oxo6fr?apikey=myapikey', {
```

Comment:
Don't you need to do a second `await` with `.json()` to get the actual document? The initial result is just the "response" object as I recall.

Reply:
(One of the reasons `apos.http` is really handy, but I'm not saying we shouldn't give `fetch`-based examples, as people will use this in non-Apostrophe-rendered web apps.)